Boyadjie/Uncover
# feat(landing): complete header

## 🤔 Why?

To start the landing page, i firstly created the header

## 💻 How?

I've work on the page under `/app/page.tsx` and created the components required for the navigation and the header

## ✅ How to validate this PR?

- Check the visual in your local project
- Check the code
- Check the responsive
- Check the tests
